runbotics/runbotics-orchestrator-ui/src/src-landing/components/CaruselButton/CaruselButton.module.scss

```diff
@@ -0,0 +1,19 @@
+.root {
+  box-shadow: 0px 16px 26px #0000000f;
+  background-color: #fff;

```

Comment:
shouldn't it be imported from the global palette? I hope that we will be doing dark mode in the future so lets make this easier in the future